pennylane/qchem/dipole.py

```diff
@@ -188,9 +189,16 @@ def fermionic_dipole(mol, cutoff=1.0e-18, core=None, active=None):
     >>>                   [3.42525091, 0.62391373, 0.1688554]], requires_grad=True)
     >>> mol = qml.qchem.Molecule(symbols, geometry, alpha=alpha)
     >>> args = [alpha]
-    >>> coeffs, ops = fermionic_dipole(mol)(*args)[2]
-    >>> ops
-    [[], [0, 0], [0, 2], [1, 1], [1, 3], [2, 0], [2, 2], [3, 1], [3, 3]]
+    >>> fermionic_dipole(mol)(*args)[2]
```

Comment:
Should the [2] be there?

```suggestion
    >>> fermionic_dipole(mol)(*args)
```

Reply:
Yes, it has three components at three directions.